min_satellites_visible: 6
min_voltage: 12.0
max_cpu_load: 90.0
